libs/lb/lb-rs/src/subscribers/search.rs

```diff
@@ -27,20 +27,38 @@ pub struct SearchIndex {
     pub tantivy_reader: IndexReader,
 }
 
-#[derive(Copy, Clone, Debug)]
+impl Serialize for SearchIndex {
```

Comment:
Not sure why serialization stuff was added here, this file is prob only for LbServer internals.

Reply:
still I may be mis-understanding but I'm fairly sure that no one will be serializing / deserializing the whole search index -- which is basically every document stored in mem